### DISCUSSION TITLE
If a request was updated, query the user if changes must be saved

### DISCUSSION THREAD

**Discussion by davinishi** [thunderclient/thunder-client-support#609] [Ideas]
Have a third option to set up the update behaviour:
1. Automatically save;
2. Don't save;
3. Query to save or discard when closing.

**Comment by rangav** [comment]
@davinishi 1 and 2 options are possible now.

Atomically save is the default option, when you run the requests it will save.

3rd option is currently not possible, have plans to implement it at later stage.
